package.json

```diff
@@ -7,9 +7,10 @@
     "lerna-changelog": "cli.js"
   },
   "scripts": {
-    "build": "babel src -d lib",
+    "build": "npm run clean && babel src --out-dir lib --ignore src/__mocks__/GithubAPI.js,src/__mocks__/Changelog.js",
```

Comment:
We don't publish Jest's mocks to npm. They are internal to Jest, I suggest you do the same.